doc/visual-programming/source/widgets/evaluate/testandscore.md

```diff
@@ -17,38 +17,42 @@ The widget tests learning algorithms. Different sampling schemes are available,
 
 The *Learner* signal has an uncommon property: it can be connected to more than one widget to test multiple learners with the same procedures.
 
-![](images/TestLearners-stamped.png)
+![](images/TestAndScore-stamped.png)
 
 1. The widget supports various sampling methods.
    - [Cross-validation](https://en.wikipedia.org/wiki/Cross-validation_\(statistics\)) splits the data into a given number of folds (usually 5 or 10). The algorithm is tested by holding out examples from one fold at a time; the model is induced from other folds and examples from the held out fold are classified. This is repeated for all the folds.
-   - **Leave-one-out** is similar, but it holds out one instance at a time, inducing the model from all others and then classifying the held out instances. This method is obviously very stable, reliable... and very slow.
+   - **Cross validation by feature** performs cross-validation but folds are defined by the selected categorical feature from meta-features.   
    - **Random sampling** randomly splits the data into the training and testing set in the given proportion (e.g. 70:30); the whole procedure is repeated for a specified number of times.
+   - **Leave-one-out** is similar, but it holds out one instance at a time, inducing the model from all others and then classifying the held out instances. This method is obviously very stable, reliable... and very slow.
    - **Test on train data** uses the whole dataset for training and then for testing. This method practically always gives wrong results.
    - **Test on test data**: the above methods use the data from *Data* signal only. To input another dataset with testing examples (for instance from another file or some data selected in another widget), we select *Separate Test Data* signal in the communication channel and select Test on test data.
 2. For classification, *Target class* can be selected at the bottom of the widget. When *Target class* is (Average over classes), methods return scores that are weighted averages over all classes. For example, in case of the classifier with 3 classes, scores are computed for class 1 as a target class, class 2 as a target class, and class 3 as a target class. Those scores are averaged with weights based on the class size to retrieve the final score.
-3. Produce a report.
-4. The widget will compute a number of performance statistics, which can be shown or hidden by right-clicking the columns:
+3. The widget will compute a number of performance statistics, which can be shown or hidden by right-clicking the columns:
```

Comment:
I think the wording here is not quite ok (this is probably some of my old stuff). Would you mind changing this to:

> The widget will compute a number of performance statistics. A few are shown by default. To see others, right-click on the header and select the desired statistic.

Or something similar.